### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

Updates readme so Indico runs 

## Fixes 

issue https://warthogs.atlassian.net/browse/WD-6708

### CODE REVIEW

**Comment by petesfrench** [comment]
Hey @mtruj013, could you have a go at following the new readme instruction please

**Comment by mtruj013** [comment]
Just to leave an update: I am still not able to run the project as I am missing SMTP information. See MM comment [here](https://chat.canonical.com/canonical/pl/rif34s34qjb8pjesce7ruynhno)

File: README.md [hunk 1]
```diff
@@ -47,17 +47,28 @@ git clone https://github.com/indico/indico.git src
 
 ```
 sudo apt-get install libsqlite3-dev
-# reinstall Python via pyenv
+```
+
+Then from inside your pyenv:
+```
+pynev install 3.9.9
```
Comment (mtruj013) [inline README.md:54]:
```suggestion
pyenv install 3.9.9
```
forgot to flag earlier, my bad

**Review by mtruj013 — APPROVED** [review]
SUCCESSSSSSS 🎉 (just a typo left)

**Pull request merged**